spec/converter_spec.rb

```diff
@@ -37,7 +37,8 @@
     it 'converts base SVG from the source directory to PNG in the destination directory' do
       setup_working_collection
       converter.preprocess("#{@support_dir}/tmp/collection")
-      converter.rasterize([Emojidex::Emoji.new(code: 'kiss')], "#{@support_dir}/tmp/collection")
+      converter.rasterize([Emojidex::Data::Emoji.new(code: 'kiss')],
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.